moveit_experimental/jog_arm/config/sia5_simulated_config.yaml

```diff
@@ -0,0 +1,34 @@
+use_gazebo: true # Whether the robot is started in a Gazebo simulation environment
+check_collisions: true # Check collisions?
+command_in_type: "unitless" # "unitless"> in the range [-1:1], as if from joystick. "speed_units"> cmds are in m/s and rad/s
+scale: # Only used if command_in_type=="unitless"
+  linear: 0.375  # Max linear velocity [m/s]
+  rotational: 0.750  # Max angular velocity [rad/s]
+  joint: 1.25  # Max joint angular/linear velocity. [m/s] or [rad/s]
+cartesian_command_in_topic: jog_arm_server/delta_jog_cmds # Topic for xyz commands
+joint_command_in_topic: jog_arm_server/joint_delta_jog_cmds # Topic for angle commands
+command_frame: base_link  # If incoming cmds have no frame, this is the default
+incoming_command_timeout: 5  # Stop jogging if X seconds elapse without a new cmd
+joint_topic: joint_states
+move_group_name: arm  # Often 'manipulator' or 'arm'
+lower_singularity_threshold: 15  # Start decelerating when the condition number hits this (close to singularity). Larger --> closer to singularity
+hard_stop_singularity_threshold: 30 # Stop when the condition number hits this. Larger --> closer to singularity
+collision_proximity_threshold: 0.03 # Start decelerating when a collision is this far [m]
+planning_frame: base_link  # The MoveIt! planning frame. Often 'base_link'
+low_pass_filter_coeff: 2.  # Larger-> more smoothing of jog commands, but more lag.
+publish_period: 0.008  # 1/Nominal publish rate [seconds]
+publish_delay: 0.005  # delay between calculation and execution start of command
+collision_check_rate: 5 # [Hz] Collision-checking can easily bog down a CPU if done too often.
+num_halt_msgs_to_publish: 4 # If 0, republish commands forever even if the robot is stationary. Otherwise, specify num. to publish
+# Publish boolean warnings to this topic
+warning_topic: jog_arm_server/warning
+joint_limit_margin: 0.1 # added as a buffer to joint limits [radians]. If moving quickly, make this larger.
+command_out_topic: sia5_controller/command
```

Comment:
I wouldn't leave the sia in here. At minimum we should put a comment in explaining how to find the correct topic for your robot.

Reply:
Ah, I see now that this is just for the sia.

Reply:
I'll add a comment about how to find it anyway. Why not